### PR TITLE
Api 7941 vv regression test suite

### PR DESCRIPTION
**Pull request by esubocz-va**
Create java based regression test suite. 
Maven build will start locally running instances of mock-mpi, mock-emis, and veteran-verification, and execute IT tests.
Maven deploy will create a docker image to run tests. See `local-docker-image` script and `veteran-verification-tests/README.md` for build and run details. Tests will only run in local, QA, Staging Lab, and Lab environments, as the tests would require PII to be run in Staging and Production - which will use real data services rather than mocks.
Related story: https://vajira.max.gov/browse/API-7941

### CODE REVIEW

File: veteran-verification-tests/src/main/java/gov/va/api/lighthouse/veteranverification/tests/SystemDefinitions.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                "/veteran_verification/"))
+        .build();
+  }
+
+  private static SystemDefinition stagingLab() {
+    return SystemDefinition.builder()
+        .veteranVerification(
+            serviceDefinition(
+                "veteran-verification",
+                "https://blue.staging_lab.lighthouse.va.gov",
```
Comment (beaugrantham-va) [inline veteran-verification-tests/src/main/java/gov/va/api/lighthouse/veteranverification/tests/SystemDefinitions.java:76]: I think this is wrong.

File: veteran-verification-tests/local-docker-image [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
```
Comment (crolarlibertyva) [inline veteran-verification-tests/local-docker-image:1]: For me this file had to be made executable with `chmod +x`

File: veteran-verification-tests/start-processes-for-integration-tests.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```
Comment (crolarlibertyva) [inline veteran-verification-tests/start-processes-for-integration-tests.sh:1]: Had to `chmod  +x` for this one too.

**Review by lukemarkwordtlibertyits — DISMISSED** [review]
This is very cool! I have confirmed the scripts are working for both `start-processes-for-integration-tests.sh` and just a normal local run.

**Review by crolarlibertyva — APPROVED** [review]

File: veteran-verification-tests/src/main/java/gov/va/api/lighthouse/veteranverification/tests/SystemDefinitions.java [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+                "/veteran_verification/"))
+        .build();
+  }
+
+  private static SystemDefinition local() {
+    return SystemDefinition.builder()
+        .attributes(attributes())
+        .icns(icns())
+        .veteranVerification(
+            serviceDefinition(
```
Comment (beaugrantham-va) [inline veteran-verification-tests/src/main/java/gov/va/api/lighthouse/veteranverification/tests/SystemDefinitions.java:118]: Is this lab definition necessary for the local environment?

**Review by beaugrantham-va — APPROVED** [review]

**Pull request merged**